packages/button/docs/Button.stories.tsx

```diff
@@ -39,13 +39,13 @@ export default {
 
 export const DefaultKaizenSiteDemo = args => <Button {...args} />
 DefaultKaizenSiteDemo.story = {
-  name: "Default (Kaizen Demo)",
+  name: "Button",
```

Comment:
Doesn't this change the embed on the site page?

Reply:
Nope, it's based on the component name rather than the story name (I double checked this incase).

There will be a mass search and replace PR once we decide on the naming conventions but for now I just updated the story display name to be less convoluted and confusing.